src/App.tsx

```diff
@@ -1,72 +1,65 @@
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { DropdownMenu } from './components/DropdownMenu';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [title, setTitle] = useState(selectedPerson?.name || '');
+
+  console.log(selectedPerson)
+
+  useEffect(() => {
+    setTitle(selectedPerson?.name || '');
+  }, [selectedPerson]);
+
+  const filteredUsers = useMemo(() => {
+    return [...peopleFromServer].filter(person => person.name.includes(title));
+  }, [title]);
+
+  const [inputFieldFocus, setInputFieldFocus] = useState(false);
+
+  function handleTitle(event: React.ChangeEvent<HTMLInputElement>) {
+    setTitle(event.target.value);
+  }
+
+  function handleInputFieldFocus() {
+    setInputFieldFocus(true);
+  }
+
+  function handleInputFieldBlur() {
+    setInputFieldFocus(false);
+    setTitle('');
```

Comment:
Clearing the title on input field blur might not be the best user experience, as it resets the search input. Consider keeping the input value to allow users to continue their search after the input field loses focus.